### PR TITLE
A beginner guide to SoftMax Regression using TensorFlow

### LINKED ISSUE #6879: A beginner guide to SoftMax Regression using TensorFlow
#### Proposed title of article
A beginner guide to SoftMax Regression using TensorFlow

#### Proposed article introduction
When compared to normal classification/regression Machine Learning problems like as house prices or Titanic survivors, this one is far more difficult to answer. Using the pixels that represent the numbers in this problem, we can figure out how to solve the riddle. There is a data collection that includes pixels at different positions on the grid of numbers that we are provided with. This is referred described as a "Deep Learning" problem since it involves a more advanced use of machine learning. When classifying the data, we will utilize a Softmax Regression model that has been SGD-optimized to aid us.

#### Key takeaways 
- Understanding what is Softmax Regression
- An easy-to-understand guide to linear regression
- Implement Softmax Regression on the MNIST handwritten digit dataset using TensorFlow library

#### Article quality
This tutorial is meant for people who are new to machine learning as well as those who are new to the Kaggle community. We will use a straightforward approach known as SoftMax regression to tackle the classification issue for handwritten digits in the MNIST dataset.

#### References
N/A

### PR DESCRIPTION
**Pull request by Wanyoike-shift**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

This closes #6879

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

**Comment by Wanyoike-shift** [comment]
Hello @lalith1403, Any update on this PR!?

**Comment by WanjaMIKE** [comment]
Merged #7477 into master
